submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+# Burning Questions
+
+*by Sir Fendi*
+
+*version 1.2*
+
+(Calculations will be double-checked)
+
+Media:
+
+* Images are the slides in a separate PowerPoint file. They are in order.
+
+* There is a preview image of the table as a PNG.
+
+All feedback is welcome! I am wondering:
+
+- Maybe more images can be added?
+
+- Are there any other posts/explorations you have seen in the past that would be cool to mention here?
+
+. . . . . . . .
+
+Factorio has a variety of fuels that can go into your burner devices, ranging from classic and modest wood, up to experimental and extreme nuclear rocket fuel. The most popular burner fuel is coal, which is the backbone of early game automation and it runs all the boilers, furnaces, vehicles, and other machines reliably until it inevitably runs low. 
+
+{% include image.html
+
+src= 'src'
+ alt='Factorio's burner fuels'
+ caption='Factorio's burner fuels.'
+
+%}
+
+In the mid game, solid fuel is introduced as the first processed fuel, made initially from crude oil. Conveniently, solid fuel is used to later make rocket fuel, which is in turn used to make nuclear fuel. These processed fuels offer noticeable advantages over coal, like having greater energy density both per unit and per stack, and an acceleration bonus when used in vehicles. More curiously, however, the late game also brings the technology to liquify good old coal and convert it into solid fuel. This raised for me the question: Does processing coal into solid fuel get more energy out of it? In other words, if you liquify 10 units of coal, make solid fuel out of the oils and gas that it yields, and then burn the solid fuel, does it provide more energy than the directly burning the coal, even with all that processing? And what if you further processed it into rocket fuel? Similar burning questions followed and snowballed into this general exploration of the energy contents of different burner fuels as well as the energy costs of converting them. 
+
+As I set off to answer these questions, a great info source was the [the wiki page for fuels](https://wiki.factorio.com/Fuel) and I approached the conversions in terms of energy produced and consumed per unit of input, rather than the production rates per building. At first I tried solving many equations with a growing list of variables, but it was much easier to convert everything into diagrams, which now summarize everything nicely. Hence, in this piece, I present some questions about different burner fuels and answer them mainly using conversion diagrams. The results provide some interesting tidbits about managing your resources and they help to complete the big picture for burner fuels.
```

Comment:
```suggestion
As I set off to answer these questions, a great info source was [the wiki page for fuels](https://wiki.factorio.com/Fuel) and I approached the conversions in terms of energy produced and consumed per unit of input, rather than the production rates per building. At first I tried solving many equations with a growing list of variables, but it was much easier to convert everything into diagrams, which now summarize everything nicely. Hence, in this piece, I present some questions about different burner fuels and answer them mainly using conversion diagrams. The results provide some interesting tidbits about managing your resources and they help to complete the big picture for burner fuels.
```